src/routes/actions.js

```diff
@@ -5,13 +5,19 @@ const context = require('../context');
  */
 class Actions {
   constructor({
-    logger, pathService, stringUtils, schemasGenerator, smartActionHook,
+    logger,
+    pathService,
+    stringUtils,
+    schemasGenerator,
+    smartActionHook,
+    smartActionHookDeserializer,
   } = context.inject()) {
     this.path = pathService;
     this.logger = logger;
     this.stringUtils = stringUtils;
     this.schemasGenerator = schemasGenerator;
     this.smartActionHook = smartActionHook;
```

Comment:
(Not related to your contribution, but having `smartActionHook` & `smartActionHookDeserializer` in the same constructor makes things unclear to me. I would say it might be a good time to rename `smartActionHook` to something a bit more explicit)

Reply:
That will be corrected in this PR #753